index.mjs

```diff
@@ -53,6 +53,9 @@ function makeId(length) {
         req.originalUrl = u;
         next();
       }
+    } else if (/^\/login/.test(o.pathname)) {
```

Comment:
`/^\/login$/`?

Reply:
$ will match with the end of the string. While our express don't end at login it has id & code as well.